templates/page.tpl.php

```diff
@@ -87,18 +87,15 @@
 </main>
 
 <footer role="contentinfo">
-  <section class="page-footer">
-    <div class="wrapper">
-      <section class="footer-top">
-        <nav>
-          <?php print render($page['footer']); ?>
-        </nav>
-      </section>
-    </div>
-  </section>
-  <section class="page-bottom">
-    <div class="wrapper">
+  <div class="wrapper">
+    <section class="footer-top">
+      <?php print render($page['footer_top']); ?>
+    </section>
+    <section class="footer-bottom">
+      <?php print render($page['footer_bottom']); ?>
```

Comment:
That's an interesting question...

We could make the base theme implement links in the footer inline by creating a template file that targeted menu blocks in the footer regions and adding the `footer-links` class to them.

However, I think that's a content decision.  I suspect it's in the kit because that's the implementation on gov.au rather than actually being part of the guide.  Thoughts?